common/src/main/java/com/codeforcommunity/enums/Grade.java

```diff
@@ -0,0 +1,36 @@
+package com.codeforcommunity.enums;
+
+public enum Grade {
```

Comment:
For the sake of consistency with our other enums, let update these too look like:

```
KINDERGARTEN("kindergarten"),
FIRST_GRADE("first_grade"),
SECOND_GRADE("second_grade"),
THIRD_GRADE("third_grade"),
FOURTH_GRADE("fourth_grade"),
FIFTH_GRADE("fifth_grade"),
SIXTH_GRADE("sixth_grade"),
FORM_ONE("form_one"),
FORM_TWO("form_two"),
FORM_THREE("form_three"),
FORM_FOUR("form_four"),
FORM_FIVE("form_five");
```